### PR TITLE
Add meeting notes: 2018-04-05

### LINKED ISSUE #287: Node.js Foundation Community Committee Meeting 2018-04-05
## Time

**UTC Thu 05-Apr-2018 16:00 (04:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Thu 05-Apr-2018 09:00 (09:00 AM) |
| US / Mountain | Thu 05-Apr-2018 10:00 (10:00 AM) |
| US / Central  | Thu 05-Apr-2018 11:00 (11:00 AM) |
| US / Eastern  | Thu 05-Apr-2018 12:00 (12:00 PM) |
| London        | Thu 05-Apr-2018 17:00 (05:00 PM) |
| Amsterdam     | Thu 05-Apr-2018 18:00 (06:00 PM) |
| Moscow        | Thu 05-Apr-2018 19:00 (07:00 PM) |
| Chennai       | Thu 05-Apr-2018 21:30 (09:30 PM) |
| Hangzhou      | Fri 06-Apr-2018 00:00 (12:00 AM) |
| Tokyo         | Fri 06-Apr-2018 01:00 (01:00 AM) |
| Sydney        | Fri 06-Apr-2018 02:00 (02:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Community%20Committee+Meeting+2018-04-05&iso=20180405T16
* or http://www.wolframalpha.com/input/?i=04PM+UTC%2C+Apr+05%2C+2018+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1wYLCX-h3a5080GinrO-RyL8mj3f9lbA7egdvIsP9MB0/edit>

## Agenda

Extracted from **cc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

- [Travel Fund approval and being careful about funds available](https://github.com/nodejs/admin/issues/99)
- [CommComm Membership: Call For Feedback on New Definitions](https://github.com/nodejs/community-committee/issues/276)
- [Define Working Group expectations and create boilerplate](https://github.com/nodejs/community-committee/issues/201)
- [Automate Meeting Participant History Document](https://github.com/nodejs/community-committee/issues/267)
- [Initiative for Nodejs Community Chapters](https://github.com/nodejs/community-committee/issues/280)

## Invited

* CommComm Members: @nodejs/community-committee

### Observers
* @Tiriel (Benjamin Zaslavsky - observer)
* @DiegoZoracKy (Diego ZoracKy - observer)
* @ZibbyKeaton (Zibby Keaton - observer)
* @Bamieh (Ahmad Bamieh - observer)
* @pchrysa (Chrysa - observer)
* @foadlab (observer)
* @codeekage (Abraham Agiri - observer)
* @Voskan (Voskan Voskanyan - observer)
* @waleedashraf (Waleed Ashraf - observer)
* @maddhruv (Dhruv Jain - observer)
* @elainechan (Elaine Chan - observer)

## Notes

The agenda comes from issues labelled with `cc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: https://zoom.us/j/793738688
* For those who just want to watch: 
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

### PR DESCRIPTION
**Pull request by JemBijoux**
Closes #287

### CODE REVIEW

File: meetings/2018-04-05.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+## Links
+
+- **Recording**: https://www.youtube.com/watch?v=Dm0rdmrxKpY
+- **GitHub Issue**: https://github.com/nodejs/community-committee/issues/287
+- **Minutes Google Doc**: https://docs.google.com/document/d/1wYLCX-h3a5080GinrO-RyL8mj3f9lbA7egdvIsP9MB0/edit
+
+## Present
+
+- @JemBijoux (Jem Bezooyen - member)
+- @mhdawson(Michael Dawson - member)
```
Comment (chowdhurian) [inline meetings/2018-04-05.md]: @ mention needs a space 😅

Comment (JemBijoux) [reply meetings/2018-04-05.md]: Good catch!

**Comment by codeekage** [comment]
@JemBijoux, Is not staying until the end of the meeting counts as absent?

**Comment by JemBijoux** [comment]
Not at all. Did I miss you? Just let me know and I can add you to the list properly.

I mentioned a few times around the start of the meeting that folks should add themselves to the list of attendees. It's difficult with zoom when the attendee list shifts up and down. I always end up with duplicates or misses, so I always appreciate if folks can add themselves.

**Comment by codeekage** [comment]
Yea... you missed me. Will take note of adding to the doc while the meeting is on. Sorry about that.

**Comment by Tiriel** [comment]
Maybe we should add a note in the meeting issue generator. I'll check if it can be done when removing myself from the observers' list.

**Comment by Tiriel** [comment]
https://github.com/mhdawson/create-node-meeting-artifacts/pull/27
cc @nodejs/community-committee  if it's ok to you

**Comment by JemBijoux** [comment]
Thanks for that Tiriel, great to have that in there.

**Comment by chowdhurian** [comment]
Merged. Thanks for keeping us on track @JemBijoux! 🙌🏽

**Pull request merged**